### ISSUE TITLE
Window not composited on GNOME wayland (flatpak)

### ISSUE THREAD

**Issue by Nine-H** [libretro/RetroArch#6606]
## Description

- Installed retroarch from flatpak on fedora27
- works fine in xorg
- no window border, can't drag or resize window in wayland
- running stable

```
[nine@x1-yoghurt ~]$ flatpak list
Ref                                             Options     
org.libretro.RetroArch/x86_64/stable            user,current
org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6 user,runtime
org.freedesktop.Platform.ffmpeg/x86_64/1.6      user,runtime
org.freedesktop.Platform/x86_64/1.6             user,runtime
````

### Expected behavior

- I expect the flatpak package to function normally in a wayland session

### Actual behavior

- I get a pseudo window that can't be interacted with from gnome
- clicking on the window (or app in fullscreen) goes "through" retroarch to the apps in the background

### Steps to reproduce the bug

1. start a wayland sesson
2. flatpak run org.libretro.Retroarch
3. click anything

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 6fc6bfb

### Environment information

- OS: fedora f27
- Compiler: GCC (6.2.0)64-bit

**Comment by hizzlekizzle** [comment]
I believe this is a duplicate of https://github.com/libretro/RetroArch/issues/4778

**Comment by Nine-H** [comment]
I think it's super, and the events go straight through the window. Could just be build flags for the flatpak because I understand there is wayland support.

**Comment by costrouc** [comment]
I have the same issue Ubuntu 17.04 with wayland

**Comment by twinaphex** [comment]
Is this still an ongoing issue? Is there anybody able to help with this issue? @Themaister You have any clues?

**Comment by Nine-H** [comment]
current flatpak is working in fedora: no window controls, but you can move retroarch with the super key and it's added correctly to the window stack.

**Comment by Sunderland93** [comment]
RetroArch does not support Client-Side Decorations, which GNOME (and Weston) needs. And GNOME developers is against SSD (Server-Side Decorations), which I plan to implement in RetroArch after #7607 is merged.

**Comment by orbea** [comment]
@Sunderland93 Is this still an issue?

**Comment by Sunderland93** [comment]
@orbea since GNOME (and Weston tough) is against SSD (Server Side Decorations), it will be permanent issue, because RetroArch doesn's support client-side window decorations (it's really hard to implement it). RetroArch already support xdg-decoration protocol - standard Wayland extension which supported by Sway (and other wlroots-based compositors), and will be implemented in KDE Plasma 5.15.

**Comment by orbea** [comment]
Thanks for the update!

**Comment by ColinKinloch** [comment]
This should be resolved as of [v1.10.0](https://github.com/libretro/RetroArch/tree/v1.10.0)
